modules/Microsoft.AAD/DomainServices/readme.md

```diff
@@ -207,7 +207,45 @@ $pfxCertificate = [System.Convert]::ToBase64String($rawCertByteStream)
 
 ## Deployment examples
 
-<h3>Example 1</h3>
+<h3>Example 1: Parameters</h3>
+
+<details>
+
+<summary>via Bicep module</summary>
+
+```bicep
+resource kv1 'Microsoft.KeyVault/vaults@2019-09-01' existing = {
+  name: 'adp-<<namePrefix>>-az-kv-x-001'
+  scope: resourceGroup('<<subscriptionId>>','validation-rg')
+}
+
+module DomainServices './Microsoft.AAD/DomainServices/deploy.bicep' = {
+  name: '${uniqueString(deployment().name)}-DomainServices'
+  params: {
+    domainName: '<<namePrefix>>.onmicrosoft.com'
```

Comment:
Question to everybody (e.g. @eriqua / @rahalan  / @MariusStorhaug): Should the parameters in the examples be sorted (e.g. as the parameters table is)?

Reply:
Turns out even without using the default PSObject but `-AsHashTable`, it is actually not ordering the item - or it is, but it's not alphabetical and there are diverse issues open for this.
To this end I started implementing a function that would recursively sort any Hashtable. The idea is that we always have a consistent output.

Reply:
It may also be interesting to sort the parameters and always move the required parameters to the top. Just a thought...

Reply:
Proposed to discuss offline